scripts/tests/calibnet_stateless_mode_check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script tests the stateless mode of a forest node
+
+source "$(dirname "$0")/harness.sh"
+
+forest_init_stateless
```

Comment:
Will this node be killed in case of an error somewhere along the way?

Reply:
Yes the detached node will be killed by `trap forest_cleanup EXIT` in `harness.sh`
`